build.sh

```diff
@@ -140,6 +124,13 @@ if [ ${CLEAN} -eq 1 -a ${BUILD_BE} -eq 0 -a ${BUILD_FE} -eq 0 -a ${BUILD_SPARK_D
     exit 1
 fi
 
+if [[ -z ${WITH_MYSQL} ]]; then
```

Comment:
Just a quick question how can we enable the build with MYSQL?

Reply:
Like this:
```
WITH_MYSQL=ON WIHT_LZO=ON sh build.sh
```

I had put it in the document of `compilation.md`